src/mediarithmics/plugins/activity-analyzer/ActivityAnalyzerBasePlugin.ts

```diff
@@ -55,8 +59,8 @@ export abstract class ActivityAnalyzerPlugin extends BasePlugin {
     return activityAnalyzerPropertyResponse.data;
   }
 
-  // Method to build an instance context
-  // To be overriden to get a cutom behavior
+  // Method to process an Activity Analysis
```

Comment:
I don't think it belongs here

instanceContextBuilder is building the isntance context, not processing the actiivty

Reply:
Hmmm really weird. It looks like reformatting the code moved the comments randomly... It has never failed me before. I'm investigating

Reply:
Maybe that it is due to the use of:
```
// Comments
```

Instead of
```
/*
* Comments
*/
```

?

Reply:
Yeah multiples single-line comments fuck up. I only have my eyes to cry now.